documentation-site/pages/components/avatar.js

```diff
@@ -0,0 +1,96 @@
+import Code from './code';
+// import {Avatar} from 'baseui/avatar';
+// import Avatar from '../../static/examples/avatar/sizes';
+
+// const code = `class ShoppingList extends React.Component {
+//   render() {
+//     return (
+//       <div className="shopping-list" style={{
+//         margin: '40px',
+//         border: '5px solid pink'
+//       }}>
+//         <h1>Shopping List for Mark</h1>
+//         <ul>
+//           <li>Instagram</li>
+//           <li>WhatsApp</li>
+//           <li>Oculus</li>
+//         </ul>
+//       </div>
+//     );
+//   }
+// }`;
+
+const code = `export default class Avatar extends React.Component<PropsT, StateT> {
```

Comment:
4) Then I saw examples from `react-live` in [here](https://react-live.kitten.sh/). It seems that you can render components in `code`, as long as you pass the entire class inside `code` as well. This is not what we want, but I wanted to test it. It gives  `Unexpected token React.Component**<**PropsT, StateT>` as error.

Reply:
Flow might not work, just pure JS.